sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOTestPipelineOptions.java

```diff
@@ -133,4 +133,29 @@
   String getMongoDBDatabaseName();
 
   void setMongoDBDatabaseName(String name);
+
+  /* HCatalog */
+  @Description("HCatalog metastore host (hostname/ip address)")
+  @Default.String("hcatalog-metastore")
+  String getHCatalogMetastoreHostName();
```

Comment:
I agree - this topic reappears from time to time so it's definitely a good moment to deal with it especially because we plan to focus on improving tests/framework now. However I think some options are generic by it's nature (eg. "numberOfRecords"). How about leaving such in IOTestPipelineOptions and move "test specific" to a another options interface (eg. HCatalogPipelineOptions in hcatalog module's tests)? 

I wouldn't touch other tests in this PR though - let's do this in separate [JIRA](https://issues.apache.org/jira/browse/BEAM-4137) issue

Reply:
Ok, I will move all my options to the ioit test. From `IOTestPipelineOptions` I'll use only `numberOfRecords`.